frontend/app/globals.css

```
@@ -1,5 +1,6 @@
 @import "tailwindcss";
 @import "tw-animate-css";
+@plugin "@tailwindcss/typography";
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@plugin`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)